test/onerror/inside-test.js

```diff
@@ -33,4 +33,24 @@ QUnit.module( "QUnit.onError", function() {
 
 		assert.strictEqual( result, true, "onError should not allow other error handlers to run" );
 	} );
+
+	QUnit.test( "Should resume test runner if test was stopped", function( assert ) {
+		assert.expect( 0 );
+
+		// This would hold up the test runner, but onError should resume it
+		assert.async();
+
+		// Avoid assertion error
+		QUnit.config.current.pushFailure = function() {};
```

Comment:
There's something screwy going on with this module in the browser, which unfortunately has been hidden because we forgot to add `onerror/inside-test.js` and `onerror/outside-test.js` to test/index.html. There may be cross-test interaction—commenting out the `onError` in _this_ test doesn't hang the runner unless I also comment out the preceding "Should call pushFailure when inside a test" test. And I think this replacement `pushFailure` should make some assertions of its own.